device/demo/rooms/test/can.esc

```diff
@@ -1,12 +1,46 @@
 :pick_up
-
 set_global i/inv_can true
 set_active can false
 
-
 :look
-
 say player "A can"
 
 :use
 set_state player hat
+
+:open
+set_global turning_can true
+> [eq can_angle 180, turning_can]
+	turn_to can 90 [eq can_angle 180]
```

Comment:
Are the postfix expressions here and in the "270" and "90" blocks necessary?

Reply:
They are left-overs from when I thought I could do this easily with postfix expressions. Removed.